projects/clamav/clamav-scanfile-fuzzer.options

```diff
@@ -0,0 +1 @@
+[libfuzzer]
```

Comment:
No need for an .options file if there are no options and we aren't copying it anywhere.

Reply:
Good to know.  I trust it isn't an issue if it exists with no options?

Reply:
I assume not. I was thinking of requesting it be removed but then I saw it isn't copied into `$OUT`, so I don't think it will have any affect at all.

Reply:
Ha! One mistake overridden by another. Outstanding. :)

Well, I look forward to seeing what results oss-fuzz brings in. Should be fun!